lib/stat_tracker.rb

```diff
@@ -94,5 +94,17 @@ def goals_per_team
     team_goals
   end
 
-  
+  def look_up_team_name(team_id)
```

Comment:
excited to implement the use of this helper methods into the methods that i have written!